### PR TITLE
add descriptions for graphql schema

### PR DESCRIPTION
**Pull request by dpopp07**
This is hype. Just go through the GraphIQL with these descriptions; it feels cool.

Descriptions are added to the schema by simply adding comments above whatever you want to describe, as you will see.

I also changed the ids to type `ID` and the `quantity` field to type `Float`.

Definitely let me know if you think any of these descriptions could be more helpful or if there's anything you would add. You are the end-user after all.

### CODE REVIEW

File: bartop-api/src/api/user/user.graphql.js [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
+
+  # User ID provided by and used to interact with Auth0
   auth0Id: String
 }
 
+# Input type containing data needed to create a new User
 input UserInput {
+
+  # # User ID provided by and used to interact with Auth0
   auth0Id: String
```
Comment (jamesvclements) [inline bartop-api/src/api/user/user.graphql.js:16]: Should this be `ID` instead of `String` as well?

Comment (jamesvclements) [reply bartop-api/src/api/user/user.graphql.js:16]: Actually I guess it is `human`-readable so `String` makes sense 🤔

Comment (dpopp07) [reply bartop-api/src/api/user/user.graphql.js:16]: Yeah I think String for sure for that

**Review by jamesvclements — APPROVED** [review]
C H E E R S 🎆

**Pull request merged**